pkg/kubectl/cmd/delete.go

```diff
@@ -123,17 +124,6 @@ func RunDelete(f *cmdutil.Factory, out io.Writer, cmd *cobra.Command, args []str
 	}
 
 	ignoreNotFound := cmdutil.GetFlagBool(cmd, "ignore-not-found")
-	if deleteAll {
```

Comment:
> but if the objects are explicitly listed by name on the command line or via --filename

In these cases, I thought kubectl would "GET" the objects one by one, rather than doing a LIST, is this not the case?